test/test-Consumer.js

```diff
@@ -989,3 +1006,44 @@ test('Consumer emits "transportclose" if Transport is closed', async () =>
 				mapConsumerIdProducerId  : {}
 			});
 }, 2000);
+
+describe('Tests which should run on fresh worker/router/transport', () => 
```

Comment:
Any test after this scenario will find the worker shut down.

In order to make this more reliable the test could verify that:

1. Creating a consumer without providing a `preferredMid` assigns certain MID (`x`), without asserting the value of `x`.
2. Creating a consumer providing `preferredMid` assigns the given MID
3. Creating a consumer without providing a `preferredMid` assigns the MID (`Number(x) + 1`)